### ISSUE TITLE
Building Data

### ISSUE THREAD

**Issue by ntarbie** [McCulloughRT/BuildingsReact#4]
Hey Ryan, wondering how you created the building data? Looks like you added it as a layer in Mapbox Studio instead of externally referencing?

**Comment by McCulloughRT** [comment]
That's correct, the data came from a variety of sources provided by the City of Portland and Metro regional government. The buildings themselves are from a shapefile from Metro, and some of the data is joined onto that then converted to geojson and finally to a tileset with tippecanoe.
